cmake/macros/macro_deal_ii_pickup_tests.cmake

```diff
@@ -81,6 +81,9 @@ MACRO(DEAL_II_PICKUP_TESTS)
   # Necessary external interpreters and programs:
   #
 
+  FIND_PACKAGE(CUDA)
+  SET(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} -std=c++11 -arch=sm_35)
```

Comment:
It is, but eventually we should find a more nice way to configure this. Many of the new fancy functionality of the Pascal GPUs require sm_60 and we should have some way of detecting this IMO.